exe/slow_query_exporter

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env ruby
+
+require 'daemons'
+require 'gelf'
+require 'optparse'
+require 'syslog'
+require 'slow_query_exporter'
+
+RUN_DIRECTORY = "/var/run/slow_query_exporter"
+PROGRESS_FILE = "#{RUN_DIRECTORY}/last_timestamp"
+
+$graylog_host = "localhost"
+$graylog_port = 12201
+$delay = 0.1
+$verbose = false
+$foreground = false
+
```

Comment:
It's useful in scripts. Prevents annoying "fun" with scoping issues, since Ruby's default variable scope is not lexical. Quick quiz: what does the following script do when run?
```ruby
#!/usr/bin/env ruby

honk = "honk"

def woop(x)
  puts "#{honk}: #{x}"
end

woop("woop")
```